<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Listes</class>
 <widget class="QWidget" name="Listes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="PB_Supprimer">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>78</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer liste</string>
   </property>
  </widget>
  <widget class="QTableView" name="TV_Liste">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_AjouterListe">
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>39</y>
     <width>78</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter Liste</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CBx_Liste">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>207</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_NouvelleListe">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>182</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Fermer">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>360</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Fermer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Selection">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tous sélectionner</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Deselection">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>370</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tous Desélectionner</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
